js/private/js_binary.bzl

```diff
@@ -258,7 +258,9 @@ def _bash_launcher(ctx, entry_point_path, log_prefix_rule_set, log_prefix_rule,
         "JS_BINARY__BINDIR": "$(BINDIR)",
         "JS_BINARY__BUILD_FILE_PATH": "$(BUILD_FILE_PATH)",
         "JS_BINARY__COMPILATION_MODE": "$(COMPILATION_MODE)",
+        "JS_BINARY__PACKAGE": ctx.label.package,
```

Comment:
again, it feels like a bit of duplication since it's pretty easy/reliable to extract from JS_BINARY__TARGET - but OTOH I guess another env variable isn't expensive and it's one fewer thing for the program to be responsible for